ShippoTesting/OrderTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections;
+
+using Shippo;
+
+
+namespace ShippoTesting {
+    [TestFixture]
+    public class OrderTest : ShippoTest {
+
+        [Test]
+        public void TestValidCreate()
+        {
+            Order testObject = OrderTest.getDefaultObject();
+            Assert.IsNotNull(testObject.ObjectId);
+        }
+
+        [Test]
+        public void testValidRetrieve()
+        {
+            Order testObject = OrderTest.getDefaultObject();
+            Order retrievedObject;
+
+            retrievedObject = apiResource.RetrieveOrder((string) testObject.ObjectId);
+            Assert.AreEqual(testObject.ObjectId, retrievedObject.ObjectId);
+        }
+
+        [Test]
+        public void testListAll()
+        {
+            Hashtable parameters = new Hashtable();
+            parameters.Add("results", "1");
+            parameters.Add("page", "1");
+
+            var parcels = apiResource.AllOrders(parameters);
+            Assert.AreNotEqual(0, parcels.Data.Count);
+        }
+
+        public static Order getDefaultObject()
+        {
+            Address addressFrom = AddressTest.getDefaultObject();
+            Address addressTo = AddressTest.getDefaultObject_2();
+            Hashtable items = new Hashtable();
+            items.Add("total_amount", "10.45");
+            items.Add("weight_unit", "kg");
+            items.Add("title", "package");
```

Comment:
Where is this `items` Hashtable used? I'm not seeing it referenced anywhere

Reply:
Added missing line items